reconcile/utils/terrascript_client.py

```diff
@@ -342,17 +342,42 @@ def populate_iam_groups(self, roles):
     def _get_aws_username(user):
         return user.get('aws_username') or user['org_username']
 
+    @staticmethod
+    def _validate_mandatory_policies(account: dict,
```

Comment:
I think we should favor `Mapping[str, Any]` over concrete implementation `dict` according to [styleguide](https://service.pages.redhat.com/dev-guidelines/docs/appsre/python-style-guide/#abstract-collection-types).

Reply:
thanks!